src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java

```diff
@@ -566,4 +566,11 @@ private static LfNetwork create(int numCC, int numSC, List<Bus> buses, List<Swit
 
         return Optional.empty();
     }
+
+    static boolean participateToBalance(LfNetworkParameters parameters, Bus b) {
```

Comment:
Can you change the method name to `participateToSlackDistribution`? It will be more coherent with the rest of the code.